docs/pages/reference/session-recording.mdx

```diff
@@ -0,0 +1,179 @@
+---
+title: Teleport Session Recording
+description: An overview of Teleport's session recording and its configuration
+---
+
+This page provides an overview of how Teleport records sessions and the various
+configuration options that apply to session recordings. The content that is
+captured depends on the type of the session.
```

Comment:
```suggestion
configuration options that apply to session recordings. 

## What session recordings capture

The content session recordings capture depends on the type of the session.
```
Since this information is pretty granular, I thought it would make sense to separate it from the intro paragraph.